core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/configure-k8s-apps-monitoring.yml

```
@@ -1,16 +1,33 @@
 ---
-- name: Set master hostname variable
-  set_fact:
-    master_hostname: "{{ groups['kubernetes_master'] | first }}"
+- name: Set facts for "classic epiphany k8s"
+  when: k8s_as_cloud_service is defined and k8s_as_cloud_service is false
```

I think this should be safer. :+1:
```suggestion
  when: k8s_as_cloud_service is defined and not k8s_as_cloud_service
```